app/retail/templates/landing/funder.html

```diff
@@ -31,14 +31,17 @@
     <div id="landing_page_wrapper">
       <div class="container-fluid header">
         {% include 'landing/funder/hero.html' %}
-        <div class="row banner--green px-3 py-4 justify-content-center font-subheader">
-          <p class="col-12 mb-2 white text-center">{% trans "DEVELOPERS! Looking for work on OSS Projects?" %}</p>
-          <div class="col-12 text-highlight-gc-purple text-center">
-            <a class="text-highlight-gc-purple" href="{% url "explorer" %}">{% trans "View Available Bounties" %}</a>
-            |
-            <a class="text-highlight-gc-purple" href="{% url "onboard" "contributor" %}">{% trans "Get Started with Gitcoin" %}</a>
-            |
-            <a class="text-highlight-gc-purple" href="/contributor">{% trans "Learn More" %}</a>
+        <div class="row banner--green px-3 py-4 font-subheader">
+          <div class="col-12 col-md-1 offset-lg-1 offset-xl-3 col-xl-1 text-center text-md-right my-auto">
+            <img src="{% static "v2/images/landing/robot-banner.svg" %}">
+          </div>
+          <div class="col-12 col-md-7 col-lg-6 col-xl-3 mt-3 mb-4 my-md-auto text-center">
+            <p class="font-title-lg mb-0">{% trans "Are you a developer or designer?" %}</p>
+            <p class="font-subheader mb-0">{% trans "Contribute to exciting OSS project and get paid!" %}</p>
+          </div>
+          <div class="col-12 col-md-4 col-xl-2 my-auto text-center">
+            <a class="button button--primary" href="{% url "explorer" %}">Get Started</a>
+            <a class="button button--primary btn-green ml-lg-2" href="/contributor">Learn More</a>
```

Comment:
Can you update the hardcoded url to use `{% url '' %}` again?

Reply:
Ah shoot ! copy paste :P